src/headings.js

```diff
@@ -12,11 +12,23 @@ import ListDropdownController from '../ui/dropdown/list/listdropdown.js';
 import ListDropdownView from '../ui/dropdown/list/listdropdownview.js';
 import Collection from '../utils/collection.js';
 
+/**
+ * The headings feature. Handles switching between different block types.
```

Comment:
Those "different block types" needs better explanation. Or even: "The headings feature. Introduces a "headings" drop-down and a command which allow switching paragraphs into headings."